proto/wechaty/puppet/mini_program.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+package wechaty.puppet;
+
+option go_package       = "github.com/wechaty/go-grpc/wechaty/puppet";
+option java_package     = "io.github.wechaty.grpc.puppet";
+option csharp_namespace = "github.wechaty.grpc.puppet";
+
+import "google/protobuf/wrappers.proto";
+
+message MiniProgramPayload {
+  google.protobuf.StringValue appid = 1;
```

Comment:
change to `optional string xxx = n;`

Reply:
We can only use `string xxx` and `optional` is not necessary because then can just be `undefined` or `''` for our payload.